lib/ansible/plugins/strategy/linear.py

```diff
@@ -261,6 +261,7 @@ def run(self, iterator, play_context):
                     # will only send this task to the first host in the list.
 
                     task.action = templar.template(task.action)
+                    task.any_errors_fatal = templar.template(task.any_errors_fatal)
```

Comment:
use post_validate, since templating leaves out other processing and checks.

Reply:
What do you mean? I did check and it is being properly updated in post_validate but it doesn't appear to be properly evaluated at this point in the strategy execution.

Reply:
sorry, for single value use `get_validated_value` which is the 'specific' version of post_validate

Reply:
got it. thanks

Reply:
I think i figured out what you were asking for, please let me know if i'm wrong